Project.toml

```diff
@@ -14,13 +14,13 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
-AxisArrays = "0.4"
-DocStringExtensions = "0.8"
+AxisArrays = "^0.4"
+DocStringExtensions = "^0.8"
```

Comment:
`DocStringExtensions` is currently 0.9.3, maybe we should add `^0.9` too? Not sure if it's still compatible but I think so...